static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -0,0 +1,32 @@
+# Caveats running DVC on Windows
+
+There can be few performance issues while running DVC on Windows. Below are some
```

Comment:
by default ... for some of these issues there are workarounds or configuration options available

Reply:
Done